lib/usd/translators/shading/fileTextureUtil.h

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright 2021 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef FILE_TEXTURE_UTIL_H
+#define FILE_TEXTURE_UTIL_H
+
+/// \file
+/// Generic utilities shared between all file texture exporters and importers
+
+#include <pxr/pxr.h>
+
+#include <string>
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+struct FileTextureUtil
```

Comment:
I'm not sure how much we expect this file to grow, but we already have https://github.com/Autodesk/maya-usd/blob/dev/lib/mayaUsd/fileio/utils/shadingUtil.h that could be a good place for these.

Reply:
Moved.